### ISSUE TITLE
Godot 4.0.1: shortcuts don't work in Windows 10

### ISSUE THREAD

**Issue by elmencho-bfg** [godotengine/godot#75355]
### Godot version

4.0.1 stable x64 without .Net

### System information

Windows 10 64bit

### Issue description

Shortcuts don't work (CTRL+key) in file Godot_v4.0.1-stable_win64.exe, but in Godot_v4.0.1-stable_console_win64.exe work.

### Steps to reproduce

CTRL+S (saving script or scene), home/end key, backspace and arrows (when renaming file or editing script), etc

### Minimal reproduction project

N/A

**Comment by viksl** [comment]
I'm in C# version on win 10 and I also have issues with saving a scene through a shortcut ctrl + s which sometimes doesn't work.
It did not work at all then I closed everything, treid to create a new project and then go back to the old one and it worked again.

I don't know what triggers it but something indeed does, I'm using the console starter so I don't think it's particularly different based on console or not.

Strangely when I stried to save a scene which I already saved before through "Scene - Save" it did not save the scene straight either but instead a new save dialog poped up for me to create a new file, when I tried to overwrite the scene itself I got an expected error that the scene is currently in use or something like that and can't be overwritten. But I have no idea why it treated an already existing and saved scene as if it was completely new.
